.changeset/calm-candles-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/app-bridge": patch
+---
+
+Feat/only one app bridge per iframe
```

Comment:
```suggestion
feat(AppBridgePlatformApp): only one app bridge per iframe
```

I would maybe add a better description to the changelog too since it is public.